packages/opentelemetry-tracing/src/types.ts

```diff
@@ -52,6 +52,12 @@ export interface TracerConfig {
 
   /** Resource associated with trace telemetry  */
   resource?: Resource;
+
+  /**
+   * Generator of trace and span IDs 
+   * The prototype IdGenerator generates random 16-byte trace ID and 8-byte span ID
```

Comment:
add default to tsdoc
```suggestion
   * The default idGenerator generates random ids
```

Reply:
Actually I initially add this to tsdoc but my mentor Anuraag suggested me to add default info here

Reply:
Not quite sure why I'm summoned here :) But it seems to be a good suggestion for cleaning up the doc, can you apply it?

Reply:
Is this conversation going to be addressed?

The PR is LGTM if the conversation has been resolved.

👍 Nice work!

Reply:
It's a minor nit so it's not blocking for me, but I would prefer to see the change made :)